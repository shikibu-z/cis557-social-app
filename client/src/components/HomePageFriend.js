import React from 'react';
import '../stylesheets/HomePageFriend.css';
import {
  Link,
} from 'react-router-dom';

function HomePageFriend(props) {
  const { id, src, name } = props;

  return (
    <Link style={{ textDecoration: 'none', color: 'black' }} to={`/chat/${id}`}>
      <div className="homepage-friend">
        <img src={src} alt="" />
        <h2>{name}</h2>
      </div>
    </Link>
  );
}

export default HomePageFriend;
