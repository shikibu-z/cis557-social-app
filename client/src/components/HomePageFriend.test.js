/*eslint-disable */
import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import HomePageFriend from './HomePageFriend';

test('HomepageFriend matches snapshot', () => {
  const component = renderer.create(<BrowserRouter><HomePageFriend /></BrowserRouter>);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
