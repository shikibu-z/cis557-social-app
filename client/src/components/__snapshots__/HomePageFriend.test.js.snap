// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomepageFriend matches snapshot 1`] = `
<a
  href="/chat/undefined"
  onClick={[Function]}
  style={
    Object {
      "color": "black",
      "textDecoration": "none",
    }
  }
>
  <div
    className="homepage-friend"
  >
    <img
      alt=""
    />
    <h2 />
  </div>
</a>
`;
